Fix generic parameter shadows outer scope generic parameter

```
INFO: From Compiling Swift module @rules_swift_package_manager~0.14.0~swift_deps~swiftpkg_swift_toolbox//:Sources_Truth:
external/rules_swift_package_manager~0.14.0~swift_deps~swiftpkg_swift_toolbox/Sources/Truth/TypeAssertions.swift:3:31: warning: generic parameter 'T' shadows generic parameter from outer scope with the same name; this is an error in Swift 6
  @discardableResult func isA<T>(
                              ^
external/rules_swift_package_manager~0.14.0~swift_deps~swiftpkg_swift_toolbox/Sources/Truth/Subject.swift:52:22: note: 'T' previously declared here
public class Subject<T>: BaseSubject {
                     ^
```

The fix is in #113.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -7,6 +7,10 @@ on:
     # Every day at 11:17 UTC.
     - cron: '17 11 * * *'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
 jobs:
   macos_build:
     runs-on: macos-13

--- a/Sources/Truth/TypeAssertions.swift
+++ b/Sources/Truth/TypeAssertions.swift
@@ -1,15 +1,15 @@
 public extension Subject {
   /// Asserts that a subject is the specified type.
-  @discardableResult func isA<T>(
-    _ expectedType: T.Type,
+  @discardableResult func isA<ET>(
+    _ expectedType: ET.Type,
     file: StaticString = #file,
     line: UInt = #line,
-    _ subjectConsumer: (Subject<T>) throws -> Void = { _ in }
+    _ subjectConsumer: (Subject<ET>) throws -> Void = { _ in }
   ) -> Self {
     guard continueAssertions else {
       return self
     }
-    guard let expected = actual as? T else {
+    guard let expected = actual as? ET else {
       fail(
         file: file,
         line: line,

